clcache.py

```diff
@@ -1528,6 +1579,8 @@ def processCompileRequest(cache, compiler, args):
     except CalledForPreprocessingError:
         printTraceStatement("Cannot cache invocation as {}: called for preprocessing".format(cmdLine))
         updateCacheStatistics(cache, Statistics.registerCallForPreprocessing)
+    except IncludeNotFoundException:
+        pass
```

Comment:
Actually, I think I rather won't do it right now. The code seems fishy and there might be a patch needed here which is worth its own PR.

This exception can apparently be raised in two situations:

1. During a cache miss, in case an include file which `cl.exe` printed (via `/showIncludes`) was deleted (or otherwise became unreadable) right after the compiler finished. Very unlikely, I guess.
2. During a cache hit, in case a manifest references a non-existent include file. In this case however, shouldn't the code rather update (i.e. rewrite) the manifest?